oregon-b/snippets-stage/deploy.yaml

```diff
@@ -38,278 +37,157 @@ spec:
+          value: "1"
         - name: NEW_RELIC_APP_NAME
-          valueFrom:
-            secretKeyRef:
-              key: new-relic-app-name
-              name: snippets-stage-v191-env
+          value: snippets-stage-oregon-b
         - name: NEW_RELIC_LICENSE_KEY
           valueFrom:
             secretKeyRef:
               key: new-relic-license-key
-              name: snippets-stage-v191-env
+              name: snippets-stage-secrets
         - name: OIDC_ENABLE
-          valueFrom:
-            secretKeyRef:
-              key: oidc-enable
-              name: snippets-stage-v191-env
+          value: "True"
         - name: OIDC_OP_AUTHORIZATION_ENDPOINT
-          valueFrom:
-            secretKeyRef:
-              key: oidc-op-authorization-endpoint
-              name: snippets-stage-v191-env
+          value: https://auth.mozilla.auth0.com/authorize
         - name: OIDC_OP_TOKEN_ENDPOINT
-          valueFrom:
-            secretKeyRef:
-              key: oidc-op-token-endpoint
-              name: snippets-stage-v191-env
+          value: https://auth.mozilla.auth0.com/oauth/token
         - name: OIDC_OP_USER_ENDPOINT
-          valueFrom:
-            secretKeyRef:
-              key: oidc-op-user-endpoint
-              name: snippets-stage-v191-env
+          value: https://auth.mozilla.auth0.com/userinfo
         - name: OIDC_RP_CLIENT_ID
           valueFrom:
             secretKeyRef:
               key: oidc-rp-client-id
-              name: snippets-stage-v191-env
+              name: snippets-stage-secrets
         - name: OIDC_RP_CLIENT_SECRET
           valueFrom:
             secretKeyRef:
               key: oidc-rp-client-secret
-              name: snippets-stage-v191-env
-        - name: PORT
-          valueFrom:
-            secretKeyRef:
-              key: port
-              name: snippets-stage-v191-env
+              name: snippets-stage-secrets
         - name: PROD_DETAILS_STORAGE
-          valueFrom:
-            secretKeyRef:
-              key: prod-details-storage
-              name: snippets-stage-v191-env
+          value: product_details.storage.PDDatabaseStorage
         - name: SECRET_KEY
           valueFrom:
             secretKeyRef:
               key: secret-key
-              name: snippets-stage-v191-env
+              name: snippets-stage-secrets
         - name: SECURE_BROWSER_XSS_FILTER
-          valueFrom:
-            secretKeyRef:
-              key: secure-browser-xss-filter
-              name: snippets-stage-v191-env
+          value: "True"
         - name: SECURE_CONTENT_TYPE_NOSNIFF
-          valueFrom:
-            secretKeyRef:
-              key: secure-content-type-nosniff
-              name: snippets-stage-v191-env
+          value: "True"
         - name: SECURE_HSTS_SECONDS
-          valueFrom:
-            secretKeyRef:
-              key: secure-hsts-seconds
-              name: snippets-stage-v191-env
+          value: "3600"
```

Comment:
This is the current value in stage, but should probably be increased to 31536000 to match prod